### ISSUE TITLE
Can dictionaries be amended?

### ISSUE THREAD

**Issue by gandalfsaxe** [streetsidesoftware/vscode-spell-checker#243]
It is possible to create global user-defined words with:
```json
 // User words to add to dictionary.  Should only be in the user settings.
    "cSpell.userWords": [],
```

But what if I want to amend a certain dictionary, e.g. the latex dictionary? Possible?

**Comment by Jason3S** [comment]
It is possible to get the same effect:

```javascript
    "cSpell.languageSettings": [
        {
            // Specify the language
            "languageId": "latex",
            // Add your words here
            "words": [
                "myownword"
             ]
        }
    ]
```

**Comment by GandalfSaxe** [comment]
Ah good. Although in practice one always uses `ctlr+.` to add words to the dictionary. It would be very tedious to have to add them manually in settings every time. I know it may be a tall order, but what if language dictionaries could be added to the IntelliSense context menu? I.e. pressing `ctrl+.` on a word with a warning would give:

```
...
Add: "" to latex directory
Add: "" to dictionary
Add: "" to folder directory
Add: "" to workspace directory
```

- if the code block you quoted in your last post is present in settings? Similarly for other languages.

**Comment by Jason3S** [comment]
Related to: #323

**Comment by Jason3S** [comment]
It is now possible to choose which dictionary you want to add a word to.

Since the way it works in V2 has changed, I'm closing this issue. If this is still an issue, please one a new issue describing the desired behavior.

Kind regards,
Jason
